### ISSUE TITLE
Compatibility Issue with RankMath

### ISSUE THREAD

**Issue by cheonmux** [wpcorner/footnotes-made-easy#14]
Hello,

First of all, thank you very much for developing this plugin. It works great overall, but I have encountered a minor issue that I wanted to bring to your attention.

I am experiencing a compatibility issue when using RankMath and the Footnote feature in the Gutenberg Editor. 

Specifically, when I insert a link to a footnote (( https://example.com)) in the answer field of the FAQ block provided by RankMath, the FAQ block itself does not display.

I hope this issue can be resolved. thank you!

**Comment by lumumbapl** [comment]
Hello. 

Thanks for bringing this up.

I have tried to reproduce the issue... I'm not sure if I have done it correctly, but I can see that the FAQ block is displaying just fine. 

Here is what I did:

1. Set up a development site with the following specifications:

- `WordPress 6.6.1`
- PHP Version 8.1.23
- Database: MySQL 8.0.16

2. Installed the latest version of **RankMath SEO plugin (Free version)**
3. Added a new post and in the post, added the **FAQ block by RankMath**.

The image below shows the live post view without the footnotes added into the answer field of the FAQ block by RankMath. 

![before_1](https://github.com/user-attachments/assets/b916072e-61f2-4b73-b6f7-a7a959df8d84)

The second image below shows when the footnotes have been added in the editor and the live view of the post. 

![After Inserting footnotes](https://github.com/user-attachments/assets/aade0000-0810-45fe-b855-24c8478cecb5)

Now, from the second image, you can see that the FAQ block is showing on the live post even after footnotes were added into the answer field of the FAQ Block by RankMath. 

I'm not sure what the issue may be... It may be as a result of plugin conflict or something. 

My first suggestion would be to try and adjust the `priority value` to a lower/bigger number. 

![priority](https://github.com/user-attachments/assets/b130a34a-79cd-41db-8909-96f875638814)

Try that and let me know how it goes. 

Regards.

Patrick L.

**Comment by cheonmux** [comment]
Hello. First of all, I apologize for the delayed response.

I've managed to resolve the issue by adjusting the "priority value" as you suggested. I use another plugin to display related posts and feedback at the bottom of my posts.

To adjust the display order of the footnotes at the bottom of the post, I had set this option to "4". This was causing the Rankmath FAQ block not to display.

After changing the option value from 4 to 11, everything is working normally now. Thank you so much!
